### ISSUE TITLE
503 error getting Terraform state in USA

### ISSUE THREAD

**Issue by billnbell3** [hashicorp/terraform#35613]
### Terraform Version

```shell
Latest version of Terraform local
```

### Terraform Configuration Files

Same N/A

### Debug Output

│ Error: Failed to request discovery document: 503 Service Unavailable

### Expected Behavior

No error!

### Actual Behavior

Error! Just today!

### Steps to Reproduce

terraform init

### Additional Context

N/A

### References

N/A

**Comment by liamcervante** [comment]
Hi @billnbell3, are you able to share which backend you are using for state storage? The 503 Service Unavailable suggests the error is originating in a remote service and Terraform is simply reporting that.

If your backend is stored in S3, for example, this error is likely to have originated from AWS. If you are integrated with Terraform Cloud, then it could be TFC that is reporting the error.

We need a bit more information about your configuration / remote setup before we can start to debug this. Thanks!

**Comment by billnbell3** [comment]
Yes it is terraform cloud that is giving 503

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by billnbell3** [comment]
```
terraform {
  # backend "s3" {
  #   bucket                  = "eks-karpenter-terraform"
  #   key                     = "dev"
  #   region                  = "us-east-1"
  #   shared_credentials_file = "~/.aws/credentials"
  # }

  cloud {
    organization = "exclusiveresorts"
    hostname = "app.terraform.io" # Optional; defaults to app.terraform.io
    workspaces {}
  }
}
```

**Comment by sebasslash** [comment]
:wave:  @billnbell3, are you still encountering this issue? It could be related to the [incident](https://status.hashicorp.com/incidents/qw34dy4q5jr8) yesterday. That status page does not mention increased errors, but have confirmed with our platform team that customers were indeed seeing an increased number of errors.

And to double check here from your message above, were you attempting to perform a state migration?

**Comment by jbardin** [comment]
Since we have not heard back in a while I'm going to close the issue. If you have any updates regarding the issue, feel free to open a new issue with the requested information. If you have more questions, you can also use [the community forum](https://discuss.hashicorp.com/c/terraform-core) where there are more people ready to help.

Thanks!
